docs/features/icons.stories.mdx

```diff
@@ -6,7 +6,7 @@ import { Meta, Intro, Icons } from '../../.storybook/components';
 
 <Intro>
 
-The icons used throughout Circuit UI come from SumUp's icon library, [`@sumup/icons`](Packages/icons), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its [documentation](Packages/icons) for usage instructions.
+The icons used throughout Circuit UI come from SumUp's icon library, [`@sumup/icons`](https://github.com/sumup-oss/circuit-ui/tree/main/packages/icons), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its documentation for usage instructions.
```

Comment:
This link works, try it [here](https://circuit.sumup.com/?path=/story/features-icons--page). The Storybook pages for the packages are generated from the package READMEs, so to view them locally, you need to run `yarn copy-docs` first. 

Could you revert this change and add the `yarn copy-docs` command to the `yarn bootstrap` script in the `package.json` file, please?

Reply:
added, but got some error on eslint on package.json, added on eslintignore